### PR TITLE
feat(fp_in_js): added basics of FP in JS

### LINKED ISSUE #17: FP in JS basics
Functional Programming In Javascript

### PR DESCRIPTION
**Pull request by braxtonchristensen**
added some examples of functional programming in javascript resolve #17

### CODE REVIEW

**Comment by t2013anurag** [comment]
@braxtonchristensen  - Can you write the sample test cases and output format?

**Comment by braxtonchristensen** [comment]
for sure, like an index.spec.js? should I assume you're going to be running the test though jest?

**Comment by t2013anurag** [comment]
Yeah i have a plan in place for that as well. I will integrate Travis soon.

File: fp_in_js/index.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+const all = xs =>
+  xs.reduce((acc, x) => acc && x, true)
+
+const first = xs =>
+  xs.reduce((acc, x) => acc)
+
+console.log(first([1,2,3]))
+
+
```
Comment (t2013anurag) [inline fp_in_js/index.js]: remove extra new line.

**Comment by t2013anurag** [comment]
@braxtonchristensen - any updates?

**Pull request merged**